cypress/page/address-step.page.ts

```diff
@@ -0,0 +1,22 @@
+class AddressStepPage {
+    private BottonLogin: string;
+    private CartNavigation: string
+
+    constructor() {
+        this.BottonLogin = '#SubmitLogin > span';
+        this.CartNavigation = '.cart_navigation span'
```

Comment:
This is referring to the proceed to checkout button, let's name this property in that way.